ras_backstage/controllers/collection_exercise_controller.py

```diff
@@ -127,3 +127,20 @@ def execute_collection_exercise(collection_exercise_id):
         raise ApiError(url, response.status_code)
 
     logger.debug('Successfully executed collection exercise', collection_exercise_id=collection_exercise_id)
+
+
+def update_collection_exercise_user_description(collection_exercise_id, user_description):
+    logger.debug('Updating collection exercise', collection_exercise_id=collection_exercise_id,
```

Comment:
Is it worth being more specific in the log? 'Updating collection exercise user description